packages/cli/src/Init.ts

```diff
@@ -236,7 +261,11 @@ export class Init implements Command {
       fs.mkdirSync(prismaFolder)
     }
 
-    fs.writeFileSync(path.join(prismaFolder, 'schema.prisma'), defaultSchema(provider))
+    if (args['--with-model']) {
+      fs.writeFileSync(path.join(prismaFolder, 'schema.prisma'), withModelSchema(provider))
```

Comment:
To avoid duplication of the `datasource` and `generator` blocks, could we re-use `defaultSchema` here and just add the model at the bottom?